templateflow/templateflow_get.py

```diff
@@ -0,0 +1,197 @@
+#!/opt/gensoft/exe/TemplateFlow_pyclient/0.8.0/venv/bin/python3.8
+
+import argparse
+import os
+import sys
+
+#---- as templateflow.api.TF_LAYOUT.root is set at import time based on default or TEMPLATE_FLOW env var
+#     and it is not writable, we have to import template flow with environnement set if we want to be
+#     abble to set cache dir from command line option
+
+import pprint 
+
+TOOL_VERSION = '0.1'
```

Comment:
why having a different version for this module?

Reply:
first itération of the "getter" tool, I will comply with the module version instead